docs/getting-started.adoc

```diff
@@ -0,0 +1,178 @@
+= Getting Started
+
+This document walks you through the development and deployment of a Node.js application to an OpenShift cluster using `odo`. It guides you on how to push new changes to your application and on other useful features of `odo`.
+
+== Developing and deploying an application to an OpenShift cluster
+
+. link:/docs/installation.md[Install the `odo` binary].
+. Log into an OpenShift cluster:
+
+* If you lack a local development cluster, we recommend using link:https://docs.openshift.org/latest/minishift/getting-started/installing.html[Minishift] to deploy a development OpenShift cluster.
++
+.. Run Minishift:
++
+----
+$ minishift start
+----
+.. Log into the OpenShift cluster:
++
+----
+$ odo login -u developer -p developer
+----
++
+[NOTE]
+====
+In order to make full use of `odo` functionality, it is recommended to enable the OpenShift link:https://docs.openshift.com/container-platform/3.11/architecture/service_catalog/index.html[service catalog]. Use `minishift` 1.30 or latest to enable this:
+
+----
+$ MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --extra-clusterup-flags "--enable=*,service-catalog,automation-service-broker,template-service-broker"
```

Comment:
instead of rewriting, just point it to `service catalog` section later in this doc

Reply:
@surajnarwade here we say 'Use `minishift` 1.30 or latest to enable this:'
In the main section on line 135, it says 1.22 or above, can you confirm what version we want to ask the user to use?

Reply:
`1.22 and above` is fine

Reply:
Fixed